packages/web-components/fast-components/src/anchored-region/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-anchored-region
```

Comment:
Should the name in the header be "FAST Anchored region"?

Reply:
this references the tag name of the component.